packages/integration-sdk-benchmark/package.json

```diff
@@ -12,7 +12,7 @@
   },
   "scripts": {
     "prebenchmark": "rm -rf .j1-integration",
-    "benchmark": "node ./src/index.js"
+    "benchmark": "for file in ./src/benchmarks/*; do yarn prebenchmark && node $file; done"
```

Comment:
Running all the benchmarks through `require`s in `index.js` leads to some performance impacts when running `async` benchmarks. It also can result in the `v8` engine crashing with out-of-memory errors for large tests.

Reply:
Looks like there is more for us to learn about the benchmarking package. I think your changes are a great for now!